docs/metrics/introduction/get-started-metrics.md

```diff
@@ -47,21 +47,6 @@ If you already have an Installed Collector, upgrade the Collector to start inges
         allowfullscreen
         />
 
-**Ingest Host Metrics by Installing a Collector through the Setup Wizard**
-
-If you do not have any installed metrics Sources, you can install one easily by using the Setup Wizard.
-
-<Iframe url="https://www.youtube.com/embed/QBTI6phfAKI"
```

Comment:
@jpipkin1 , have you checked with the Training Team about taking this video off YouTube?

Reply:
No I haven't, @kimsauce. But the video is still good until Setup Wizard is removed from the UI. I suppose I should tell them to take it down then when we do [DOCS-205](https://sumologic.atlassian.net/browse/DOCS-205).